src/hyp3_isce2/process.py

```diff
@@ -5,12 +5,22 @@
 import logging
 import os
 import subprocess
-from argparse import ArgumentDefaultsHelpFormatter, ArgumentParser
+from argparse import (
+    ArgumentDefaultsHelpFormatter,
+    ArgumentParser
+)
```

Comment:
Black did this?

Reply:
No, this was done with [`isort`](https://pycqa.github.io/isort/index.html). I had my line length for this tool set to 88, thus the wrapping.